docs/reference.md

```diff
@@ -1,229 +1,94 @@
+| [`Facebook\\GraphNodes\\GraphPage`](reference/GraphNode.md#graphpage-instance-methods)  | A collection that represents a Page node. |
+| [`Facebook\\GraphNodes\\GraphPicture`](reference/GraphNode.md#graphpicture-instance-methods)  | A collection that represents a Picture node. |
+| [`Facebook\\GraphNodes\\GraphUser`](reference/GraphNode.md#graphuser-instance-methods)  | A collection that represents a User node. |
+
+
+# File Uploads
 
 These are entities that represent files to be uploaded with a Graph request.
 
-%FB(devsite:markdown-wiki:table {
-  columns: ['Class name','Description',],
-  rows: [
-    [
-      '[`Facebook\\FileUpload\\FacebookFile`](/docs/php/FacebookFile)',
-      'Represents a generic file to be uploaded to the Graph API.',
-    ],
-    [
-      '[`Facebook\\FileUpload\\FacebookVideo`](/docs/php/FacebookVideo)',
-      'Represents a video file to be uploaded to the Graph API.',
-    ],
-  ],
-})
-</card>
-
-<card>
-# Extensibility {#extensibility}
+| Class name  | Description |
+| ------------- | ------------- |
+| [`Facebook\\FileUpload\\FacebookFile`](reference/FacebookFile.md) | Represents a generic file to be uploaded to the Graph API.  |
+| [`Facebook\\FileUpload\\FacebookVideo`](reference/FacebookVideo.md)  | Represents a video file to be uploaded to the Graph API.  |
+
+# Extensibility
 
 You can overwrite certain functionality of the SDK by coding to an interface and injecting an instance of your custom functionality.
 
-%FB(devsite:markdown-wiki:table {
-  columns: ['Interface name','Description',],
-  rows: [
-    [
-      '`Facebook\\HttpClients\\ FacebookHttpClientInterface`',
-      'An interface to code your own HTTP client implementation.',
-    ],
-    [
-      '`Facebook\\Http\\GraphRawResponse`',
-      'An entity that is returned from an instance of a `FacebookHttpClientInterface` that represents a raw HTTP response from the Graph API.',
-    ],
-    [
-      '[`Facebook\\PersistentData\\PersistentDataInterface`](/docs/php/PersistentDataInterface)',
-      'An interface to code your own persistent data storage implementation.',
-    ],
-    [
-      '[`Facebook\\Url\\UrlDetectionInterface`](/docs/php/UrlDetectionInterface)',
-      'An interface to code your own URL detection logic.',
-    ],
-    [
-      '[`Facebook\\PseudoRandomString\\ PseudoRandomStringGeneratorInterface`](/docs/php/PseudoRandomStringGeneratorInterface)',
-      'An interface to code your own cryptographically secure pseudo-random string generator.',
-    ],
-  ],
-})
-</card>
+| Interface name  | Description |
+| ------------- | ------------- |
+| `Facebook\\HttpClients\\ FacebookHttpClientInterface` | An interface to code your own HTTP client implementation.  |
+| `Facebook\\Http\\GraphRawResponse`  | An entity that is returned from an instance of a `FacebookHttpClientInterface` that represents a raw HTTP response from the Graph API.  |
```

Comment:
Where should these link to? I couldn't find the corresponding docs under /reference, as I did with the other links.